module/src/main/java/org/mule/extension/spring/api/security/PreAuthenticatedAuthenticationProvider.java

```diff
@@ -1,26 +0,0 @@
-/*
```

Comment:
Even if this class was not used, being in the API package it was previously exposed to users. Not sure we can remove it.